app/components/process-execution-handler-rows/component.js

```diff
@@ -1,9 +1,16 @@
 import Ember from 'ember';
 
 export default Ember.Component.extend({
+  application: Ember.computed.alias('applicationController'),
```

Comment:
Application is auto-injected into all components with `app/initializers/inject-application'